Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -145,9 +145,10 @@
     secret: true
     litDamageBonus:
         types:
-            Slash: 7.5
```

Comment:
I suggest adding a comment to specify that we changed this basegame file
`# DeltaV - Modified values to fit the ESword` or something